deepchem/models/chemberta.py

```diff
@@ -0,0 +1,83 @@
+import torch
+import torch.nn as nn
+from torch.nn import CrossEntropyLoss, MSELoss
+from transformers.modeling_roberta import (
+    ROBERTA_PRETRAINED_MODEL_ARCHIVE_LIST,
+    BertPreTrainedModel,
+    RobertaClassificationHead,
+    RobertaConfig,
+    RobertaModel,
+)
+
+
+class ChemBERTaforSequenceClassification(BertPreTrainedModel):
+    r"""
```

Comment:
Can you convert this docstring over into Numpy style? (https://numpydoc.readthedocs.io/en/latest/format.html)

Reply:
will do